### PR TITLE
HAHA I WIN

### PR DESCRIPTION
**Pull request by Jeff-Moreau**
All fixed

### CODE REVIEW

File: Source/SteveSloth/Public/MyEnemyStateComponent.h [hunk 1]
```diff
@@ -29,7 +29,8 @@ enum EStates
 	RangedAttack,
 	Frozen,
 	Confused,
-	Die
+	Die,
+	Test
```
Comment (Saretzky) [inline Source/SteveSloth/Public/MyEnemyStateComponent.h:33]: get rid of test

**Review by Saretzky — CHANGES_REQUESTED** [review]
I assume we need to add the koala stuff for all enemies

**Review by Guysus — APPROVED** [review]
good job finding and fixing the bugs

File: Source/SteveSloth/Public/MyEnemyStateComponent.h (hunk 1 above)
Comment (Saretzky) [reply Source/SteveSloth/Public/MyEnemyStateComponent.h:33]: Once this is done you're good to go

**Pull request merged**